qemu/tests/cfg/balloon_service.cfg

```diff
@@ -27,7 +27,7 @@
     polling_interval = 2
     polling_sleep_time = 10
     free_mem_cmd = wmic os get FreePhysicalMemory
-    ratio = 0.5
+    check_mem_ratio = 0.1
```

Comment:
check_mem_ratio has a default value in the case.
Of course it's better to define it in the cfg.
        check_mem_ratio = float(params.get("check_mem_ratio", 0.1))